stixcore/products/product.py

```diff
@@ -580,21 +580,20 @@ def __add__(self, other):
         other_data['old_index'] = [f"o{i}" for i in other_data['control_index']]
         self_data['old_index'] = [f"s{i}" for i in self_data['control_index']]
 
-        control = vstack((self_control, other_control))
+        if (self.service_type, self.service_subtype) == (3, 25):
+            self_data['time'] = SCETime(self_control['scet_coarse'], self_control['scet_fine'])
+            other_data['time'] = SCETime(other_control['scet_coarse'], other_control['scet_fine'])
 
         logger.debug('len self: %d, len other %d', len(self_data), len(other_data))
-
-        data = (vstack((self_data, other_data))
-                if self.scet_timerange.start <= other.scet_timerange.start
-                else vstack((other_data, self_data)))
-
+        control = vstack((self_control, other_control))
+        data = vstack((self_data, other_data))
```

Comment:
It's sorted later on line 589 so shouldn't matter which way it is stacked

Reply:
ok